Source/DafnyRuntime/DafnyRuntimeJava/src/main/java/dafny/DafnySequence.java

```diff
@@ -396,10 +396,17 @@ public String verbatimString(){
 final class ArrayDafnySequence<T> extends NonLazyDafnySequence<T> {
     // not T[] because generics and arrays don't mix
     private Object[] seq;
+    @SuppressWarnings("unused")
+    private boolean unsafe; // for debugging purposes
```

Comment:
I'd actually suggest an `isUnsafe()` method on the interface (or perhaps something more precise) which you can call in a unit test.